training/scip/intro-linux-aalto-computing.rst

```diff
@@ -103,8 +102,26 @@ Preparation
 
 **Software installation:**
 
-* It could be useful to install a SSH client, but we have a backup
-  plan if you don't. `Here some options for installing BASH on your computer <https://coderefinery.github.io/installation/bash/>`__.
+* You will need the BASH shell
+
+  * Linux: Open the Terminal application and type: ``bash``
+  * MacOS: Open the Terminal application and type: ``bash``
+  * Windows:
+
+    * If you have an Aalto account, download `PuTTY <https://www.putty.org/>`__
+      and use it to connect to ``kosh.org.aalto.fi`` (see this `screenshot
```

Comment:
```suggestion
      and use it to connect to ``kosh.aalto.fi`` (see this `screenshot
```
Normally I say kosh.aalto.fi because it's shorter and easier to remember (though inconsistend with other computers...)